sdk/storage/azure-storage-file/apireview/azure/storage/file/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+# Stubs for azure.storage.file (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._shared.models import AccountPermissions as AccountPermissions, LocationMode as LocationMode, ResourceTypes as ResourceTypes, StorageErrorCode as StorageErrorCode
+from ._shared.policies import ExponentialRetry as ExponentialRetry, LinearRetry as LinearRetry, NoRetry as NoRetry
+from .directory_client import DirectoryClient as DirectoryClient
+from .file_client import FileClient as FileClient
+from .file_service_client import FileServiceClient as FileServiceClient
+from .models import AccessPolicy as AccessPolicy, ContentSettings as ContentSettings, CorsRule as CorsRule, DirectoryProperties as DirectoryProperties, DirectoryPropertiesPaged as DirectoryPropertiesPaged, FilePermissions as FilePermissions, FileProperties as FileProperties, Handle as Handle, HandlesPaged as HandlesPaged, Metrics as Metrics, NTFSAttributes as NTFSAttributes, RetentionPolicy as RetentionPolicy, SharePermissions as SharePermissions, ShareProperties as ShareProperties, SharePropertiesPaged as SharePropertiesPaged
```

Comment:
Do we want to expose the Paged model? That's return type, if we document that as "iterator of T", exposing the model is actually not necessary.
However, this classes have a few public atttributes, that I don't know if they are public on purpose or not

Reply:
+1 As a user I would be much happier seeing "iterator of T" and not some obscure sounding object I don't know anything about

Reply:
#7800